src/api_server/Cargo.toml

```diff
@@ -6,10 +6,10 @@ edition = "2021"
 license = "Apache-2.0"
 
 [dependencies]
-libc = ">=0.2.39"
-serde = { version = ">=1.0.27", features = ["derive"] }
-serde_derive = ">=1.0.27"
-serde_json = ">=1.0.9"
+libc = "0.2.117"
```

Comment:
Should we just specify `major.minor` here and below to avoid churn?

Reply:
Though it may not work if we /really/ require `>=0.2.39`. Wouln't work for all cases.

Reply:
The difference between specifying `libc = "0.2"` and `libc = "0.2.117"` is that the latter disallows patch versions less than 117, whereas the former allows all patch releases of 0.2. 

I do not know if Firecracker requires any minimal patch releases (e.g. something like performance improvements could be done in patch releases, and firecracker might implicitly depend on those through our performance tests), the versions specified in this PR are pulled from `Cargo.lock` (e.g. are exactly the versions that `cargo build` was already using prior to this PR). 

I agree that the currently specified patch versions are somewhat arbitrary (in the end, they are just the newest patch versions that happened to be available whenever `Cargo.lock` got last updated). I'd be in favor of removing them for crates where firecracker does not depend on a specific minimal patch version, but since I do not know for which crates this is the case, I think that change if outside the scope of this PR.